### DISCUSSION TITLE
Wino Mail clone on microsoft store

### DISCUSSION THREAD

**Discussion by Thane5** [bkaankose/Wino-Mail#479] [General]
Hey, looks like someone "forked" this project and casually released it in the microsoft store. Is that allowed?
https://apps.microsoft.com/detail/9nh5bz1jst2s

**Comment by bkaankose** [comment]
Partially yes. Wino's license allow you to change the code and re-distribute as long as you open-source your own code. However, the illegal thing they do in this project is to use Wino's API keys for Outlook and Gmail synchronization. These guys have reached out to me in the past and I know what they are doing. If they change their API keys, they can do it, but with same keys it's illegal.

I have reported the app multiple times already but there were no action from the Microsoft side. I will escalate the situation to official channels to speed up this process in the upcoming days.

**Comment by Fairbanks357** [comment]
Looks like "Mail Universal" has been removed from the Microsoft Store.
